src/udiskslinuxmanager.c

```diff
@@ -424,16 +433,6 @@ handle_loop_setup (UDisksManager          *object,
                  loop_device,
                  path);
 
-  /* Update the udisks loop state file (/run/udisks2/loop) with information
-     about the new loop device created by us. We need to manually trigger
-     uevent for the device after this to update Loop interface properties. */
-  udisks_state_add_loop (udisks_daemon_get_state (manager->daemon),
-                        loop_device,
-                        path,
-                        fd_statbuf_valid ? fd_statbuf.st_dev : 0,
-                        caller_uid);
-  udisks_linux_block_object_trigger_uevent (UDISKS_LINUX_BLOCK_OBJECT (loop_object));
-
```

Comment:
What about this uevent trigger? Is it no longer needed?

Reply:
The uevent trigger was needed to update the property value after creating the loop object. The state file should now be created before creating the object so the value should be correct and we don't need to update it later.